tools/testsys/Test.toml.example

```diff
@@ -0,0 +1,99 @@
+# This is an example testing configuration for TestSys, the tool that
+# is used to validate Bottlerocket builds.
+
+# This section contains configuration details for all testing
+[test]
+# The repo from `Infra.toml` that should be used for testing. It may be
+# useful to create a repo in `Infra.toml` that contains the infrastructure
+# needed for testing
+repo = "default"
+# The registry containing alternate TestSys agent images
+testsys-images-registry = "public.ecr.aws/bottlerocket-test-system"
+
+# The URI for the EKS resource agent that should be used.
+# An individual agent's provided URI will be used even if `testsys-images-registry` is present.
+eks-resource-agent-image = "public.ecr.aws/bottlerocket-test-system/eks_resource_agent:v0.0.2"
+
+# The test configuration is broken up into different levels depending on the variant.
+# Configurations will have the following precedence:
+# Arguments from the command line
+# Environment variables from the command line
+# '<VARIANT>' configuration in `Test.toml`
+# '<FAMILY>-<FLAVOR>' configuration in `Test.toml`
+# '<FAMILY>' configuration in `Test.toml`
+# '<PLATFORM>-<FLAVOR>' configuration in `Test.toml`
```

Comment:
TODO - How do you distinguish between a variant and `<PLATFORM>-<FLAVOR>`. For example, if my platform is `p` and my flavor is `f`, then `p-f` could be a platform-flavor or it could be a variant name. Should a different delimiter be used for these partial matches?

Reply:
If that happens then it will treat the `foo-bar` table as being variant level configuration.